docs/roman/stpipe/config_asdf.rst

```diff
@@ -173,14 +177,14 @@ step is executed.
 Remember that parameter values can come from numerous sources. Refer to
 :ref:`Parameter Precedence` for a full listing of how parameters can be set.
 
-From the ``JumpStep`` example, if all that needed to change is the
-``rejection_threshold`` parameter with a setting of ``80.0``,
+From the ``SourceCatalogStep`` example, if all that needed to change is the
+``aperture_ee1`` parameter with a setting of ``30.0``,
```

Comment:
```suggestion
``aperture_ee1`` parameter with a setting of ``30.0``,
```
`aperture_ee1` is an integer. Attempting to use `30.0` results in an error:
`ValueError: Config parameter 'aperture_ee1': the value "30.0" is of the wrong type.`. Please change this and the other uses to `30`.